docs/blog/es-tooling.md

```diff
@@ -10,43 +10,43 @@ head:
       content: website
   - - meta
     - property: og:title
-      content: ES tooling - community lead e18e-focused tooling
+      content: ES tooling – community-led, e18e-focused tooling
   - - meta
     - property: og:url
       content: https://e18e.dev/blog/es-tooling
   - - meta
     - property: og:description
-      content: ES Tooling - a community lead, e18e-focused collection of tools for the ecosystem
+      content: ES Tooling, a community-led, e18e-focused collection of tools for the ecosystem
 ---
 
 # ES Tooling
 
 _July 22, 2024_
 
-The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and the creation of many community lead tools.
+The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and the creation of many community-led tools.
 
 A notable example of this is the [ES Tooling](https://github.com/orgs/es-tooling/) community GitHub organisation.
 
 If you haven't yet encountered e18e, read more in the [guide](https://e18e.dev/guide/#why-e18e) and [join the community](https://chat.e18e.dev/).
 
 ## A home for e18e-focused tools
 
-Over the last few months, many of us have collaborated and connected through the e18e community. The ES tooling organisation is a result of that - a place for us to store and own forks, tools, and other code bases we're working on together.
+Over the last few months, many of us have collaborated and connected through the e18e community. The ES tooling organisation is a result of that: a place for us to store and own forks, tools, and other codebases we're working on together.
```

Comment:
I don't think we need to change the emdash

Reply:
En-dashes imply more conceptual separation, such as a jump in thought. Colons precede an explanation with more information. This may seem like nit picking, but this is what professional copy editing looks like.